src/rttr/detail/variant/variant_data_policy.h

```diff
@@ -652,7 +652,7 @@ struct RTTR_API variant_data_policy_empty
             }
             case variant_policy_operation::GET_VALUE:
             {
-                arg.get_value<void*>() = nullptr;
+                arg.get_value<const void*>() = nullptr;
```

Comment:
was that wrong before? and did you already had a get_value function before but not in the interface in `variant` (I suppose `arg` is a variant?)

Reply:
I go down with a `const void*`, that's why I adjusted it.